integration/terraform/ecs/linux/README.md

```diff
@@ -0,0 +1,14 @@
+Running integration tests
+=========================
+
+# Run tests in your AWS account
+By running `terraform apply -auto-approve -lock=false -var="test_dir=../../../test/ecs/ecs_metadata/"`, 
+you are agree to setup the following resources:
+* 1 IAM Task Role and 1 Execution Task Role (similar to [these IAM Roles](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/deploy_servicelens_CloudWatch_agent_deploy_ECS.html))
+* 2 SSM Parameter Store
+* 2 Task Definitions and 2 Services for those task definitions
+* 1 Security group which allows all inbound and outbound traffics.
```

Comment:
Why do we need to allow **all** inbound and outbound traffic? It would be better if we know for sure what ports we need.

Reply:
Its better to allow all traffics since this is a common terraform script for ECS test, including EC2 or Fargate. To be more specifically, I would not constraint the port since there would be extra apps with multiple ports and managing with port go to which test case is a pain though.